beans/src/main/resources/org/ldaptive/beans/spring/spring-ext.xsd

```diff
@@ -109,6 +109,7 @@
       <xsd:extension base="baseAuthenticatorType">
         <xsd:attribute name="baseDn" type="xsd:string" use="required"/>
         <xsd:attribute name="userFilter" type="xsd:string" use="required"/>
+        <xsd:attribute name="passwordAttribute" type="xsd:string" use="optional"/>
```

Comment:
Were you just making this change for the sake of completeness?
None of the bean definition parsers currently support the CompareAuthenticationHandler.
It could be added as part of this PR, but it will require the implementation of a new authenticator type in the schema.